practices/runbook.md

```diff
@@ -0,0 +1,75 @@
+---
+slug: runbook
+aspect: processes
+stages:
+- development
+short_description: A runbook is a set of standardized documents that describe how to run a computer system. It typically contains a walkthrough how to start, stop, debug and supervise the system.
+tags:
+- runbook
+- documentation
+keywords:
+- runbook
+- playbook
+- procedure
+- documentation
+- decision
+- timeline
+- command
+- operation
+- task
+---
+
+# Runbook
+
+## What Is a Runbook
+
+A runbook (also referred to as a playbook) is a compilation of procedures and operations that describe how to run a computer system or network. Runbook typically contains procedures for starting, stopping, debugging and supervising the system. It is typically created by technical writers and it often contains step-by-step decision trees (often with a timeline):
+
+![Decision Tree](/files/runbook.png)
```

Comment:
The image is missing - see https://staging.developerexperience.io/practices/runbook